change/@microsoft-teams-js-ac50d7db-93a5-4b32-9b41-acdc7acdea59.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Adding missing HostClientTypes.",
```

Comment:
Can you please make this comment more specific and use past tense and backtick quotes around types? E.g.,

"Added missing \`HostClientType\` values so correct \`Runtime\` is generated for \`teams.fullTrust.joinedTeams\` and \`webStorage\` capabilities."

Reply:
Resolved in latest commit. 😃